sample_templates/emr.yml.j2

```diff
@@ -0,0 +1,73 @@
+Auth:
+  region: "{{ region }}"
+Metrics:
+{% for emr in resources %}
+{% for metric_name in (
+    'AppsCompleted',
```

Comment:
You can put this list under `MetricName`, and drop the `for` loop. It won't make any functional difference (i.e. fewer API calls), but your config will be 1/28th as verbose.